RUMM-1509: Better guess for the action name in case of automatic user interactions tracking (#93)
DataDog/dd-sdk-reactnative: README.md

// Once SDK is initialized you need to setup view tracking to be able to see data in the RUM Dashboard.
```

### User interactions tracking

> If user interactions tracking is enabled as in the code example above, SDK will traverse up components hierarchy starting from the component which received a tap, looking for `dd-action-name` property. Once found, it will be used as a name for the action reported.

Reviewer: Nitpicking: you need an empty line after a title

// Once SDK is initialized you need to setup view tracking to be able to see data in the RUM Dashboard.
```

### User interactions tracking

> If user interactions tracking is enabled as in the code example above, SDK will traverse up components hierarchy starting from the component which received a tap, looking for `dd-action-name` property. Once found, it will be used as a name for the action reported.

Reviewer 2: ```suggestion
If user interactions tracking is enabled as in the code example above, the SDK traverses up the hierarchy of components starting from the component that received a tap, looking for `dd-action-name` property. Once found, it is used as a name for the action reported.
```

> Alternatively you can use `accessibilityLabel` element property to give tap action the name, otherwise element type will be reported. You can check our sample app for the usage examples.

Reviewer 2: ```suggestion
Alternatively, you can use the `accessibilityLabel` element property to give the tap action a name; otherwise, the element type is reported. You can check the sample app for usage examples.
```
